### PR TITLE
fix congress message spec

### PR DESCRIPTION
**Pull request by jparr**
added a helper for better delayed job error reporting

### CODE REVIEW

File: app/models/congress_message.rb [hunk 1]
```diff
@@ -69,8 +69,7 @@ def background_submit(test: false)
     return unless valid?
 
     @forms.each do |f|
-      f.delay.fill(attributes_for(f.bioguide_id), campaign.campaign_tag,
-                   test: test)
+      f.delay.fill(attributes_for(f.bioguide_id), campaign.campaign_tag, test)
```
Comment (jparr) [inline app/models/congress_message.rb:72]: Looks like rubocop 'fixed' this here: https://github.com/EFForg/action-center-platform/commit/9ecfc7d2fcd031ca70c546acc23c36a60ab355b6

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -30,6 +30,12 @@
     mocks.verify_partial_doubles = true
   end
 
+  config.around(:each, :run_delayed_jobs) do |example|
```
Comment (jparr) [inline spec/spec_helper.rb:33]: added this nice helper. it runs the jobs synchronously so you get error reporting inline with the test/logging output

File: app/models/congress_message.rb (hunk 1 above)
Comment (jparr) [reply app/models/congress_message.rb:72]: I need to add something so rubocop doesnt make the same mistake again

Comment (jparr) [reply app/models/congress_message.rb:72]: I got rubocop running and it did not try to fix this again so I think its good to go!

**Review by esoterik — APPROVED** [review]

**Pull request merged**